README.md

```diff
@@ -109,8 +121,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
-
+AbrahamFSM.JPG
```

Comment:
Only small problem is that an extra state crept in: you can combine S1 and S2

![image](https://user-images.githubusercontent.com/10648/78703509-ea23ce80-78cf-11ea-961d-6207022297ed.png)

# -2